queries.sql

```diff
@@ -38,4 +38,81 @@ WHERE name != 'Gabumon'
 /*  Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
 
 SELECT * FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 AND weight_kg = 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 AND weight_kg = 17.3;
+
+-- DAY 2 --
+
+/* Inside a transaction update the animals table by setting 
+the species column to unspecified. Verify that change was 
+made. Then roll back the change and verify that species 
+columns went back to the state before tranasction. */
+
+BEGIN;
+	UPDATE animals
+	SET species = 'unspecified';
+ROLLBACK;
```

Comment:
* I suggest you properly state these insertions `Inside a transaction update the animal's table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.`  
* You only started the transaction, and made the change (` UPDATE animals SET species = 'unspecified';`) but you didn't verify that change was made and undone for example:  ` SELECT species from animals;` 
  ` ROLLBACK;` this verifies change was made.
Then
  ` SELECT species from animals; ` 
`